File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/session5/p3.c
//         1
//      1  2
//   1  2  3
//1  2  3  4

#include<stdio.h>
void main(){
	int row;
	printf("enter the row:\n");
	scanf("%d",&row);

	for(int i=1;i<=row;i++){
		int num=1;
		for(int sp=row;sp>i;sp--){
			printf(" \t");
		}

		for(int j=1;j<=i;j++){
			printf("%d\t",num);
			num++;
		}
		printf("\n");
	}
}

File: java-main/java-main/JavaDSA_Super-X/Assi_4/p3.java
import java.util.Scanner;

class Demo {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a positive integer: ");
        int num = scanner.nextInt();
        long factorial = 1;
	      if (num < 0) {
            System.out.println("Factorial is not defined for negative numbers.");
        } else if (num == 0) {
            System.out.println("The factorial of 0 is 1");
        } else {
           for (int i = 1; i <= num; i++) {
                factorial *= i;
            }
            System.out.println("The factorial of " + num + " is " + factorial);
        }

        scanner.close();
    }
}

